src/ray/object_manager/object_directory.cc

```diff
@@ -16,6 +16,9 @@ namespace {
 void UpdateObjectLocations(const std::vector<ObjectTableDataT> &location_history,
                            const ray::gcs::ClientTable &client_table,
                            std::unordered_set<ClientID> *client_ids,
+                           bool *inline_object_flag,
```

Comment:
Update the docstring for this function to describe the new arguments.

Reply:
Sorry where is the docstring?

Reply:
Ah, it's the [line](https://github.com/ray-project/ray/blob/master/src/ray/object_manager/object_directory.cc#L11) above this function. It's not a formal docstring since this is just a helper function, but still good to document!